dvc/output.py

```diff
@@ -204,6 +204,28 @@ def load_from_pipeline(stage, data, typ="outs"):
         )
 
 
+def split_file_meta_from_cloud(entry: Dict) -> Dict:
```

Comment:
I went with the way of splitting metadata from cloud during dump, and merging during load. It seemed that supporting multiple remotes or splitting cloud metadata would take too much effort, and since we don't want to support multiple remotes at this point but only metadata, I think this approach is good enough.